src/quilla/common/enums.py

```diff
@@ -38,6 +38,7 @@ class XPathValidationStates(ValidationStates):
     NOT_HAS_ATTRIBUTE = 'NotHasAttribute'
     ATTRIBUTE_HAS_VALUE = 'AttributeHasValue'
     NOT_ATTRIBUTE_HAS_VALUE = 'NotAttributeHasValue'
```

Comment:
How about "ATTRIBUTE_HAS_NO_VALUE"

Reply:
I can for sure make that change, I've just been using the convention of adding "not" before the state making it its inverse. Would you prefer the slightly more descriptive name, or the convention?

Reply:
I think "NOT_ATTRIBUTE_HAS_VALUE" may be misunderstood as "no attribute has value for all the existing attributes" I definitely prefer a more accurate naming close to natural expression.

Reply:
Resolving this due to comment chain from above, will add issue to tackle that